kratos/includes/model_part_io.h

```diff
@@ -356,6 +356,9 @@ class KRATOS_API(KRATOS_CORE) ModelPartIO : public IO
     void ReadElementalDataBlock(ElementsContainerType& rThisElements);
     
     void WriteElementalDataBlock(ElementsContainerType& rThisElements);
+    
+    template<class TVariableType>
+    void WriteElementalDataBlock(ElementsContainerType& rThisElements, TVariableType& rVariable);
```

Comment:
If TVariableType is an input parameter, it should be const.

Reply:
You are right @msandre, should be const reference.